core/src/server/rpc_module.rs

```diff
@@ -681,27 +680,17 @@ impl<Context: Send + Sync + 'static> RpcModule<Context> {
 								params,
 								id
 							);
-							let err =
-								to_json_raw_value(&"Invalid subscription ID type, must be Integer or String").ok();
-							return sink.send_error(id, invalid_subscription_err(err.as_deref()));
+							return sink.send_response(id, false);
 						}
 					};
 					let sub_id = sub_id.into_owned();
 
-					if subscribers
+					let result = subscribers
 						.lock()
 						.remove(&SubscriptionKey { conn_id: conn.conn_id, sub_id: sub_id.clone() })
-						.is_some()
-					{
-						sink.send_response(id, "Unsubscribed")
-					} else {
-						let err = to_json_raw_value(&format!(
-							"Invalid subscription ID={}",
```

Comment:
The only downside to this PR is that we loose this information (I think?). 

With this PR, when an unsubscribe call fails, what do users get back?

Reply:
yeah, unfortunately they will just receive a `bool` set to false and then the client(s) have to figure it on their own :(

Reply:
Let's document this properly in the macro docs (and on this method): "When attempting to unsubscribe with an unknown subscription ID this call will return `false`" or something along those lines?

Reply:
I guess it's better to document that we implement the `ethereum pubsub specification` when we refer to subscriptions and it's possible to provide custom subscription IDs but that's it.

Sure, we could add some documentation to both `RpcModule::register_subscription` and the macros.